packages/p2p/src/components/buy-sell/__test__/buy-sell-table.spec.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { isDesktop } from '@deriv/shared';
+import { useStores } from 'Stores';
+import BuySellTable from '../buy-sell-table.jsx';
+
+jest.mock('Stores', () => ({
+    ...jest.requireActual('Stores'),
+    useStores: jest.fn(),
+}));
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Loading: () => <div>Loading</div>,
+}));
+
+jest.mock('Components/table/table-error.jsx', () => ({
+    ...jest.requireActual('Components/table/table-error.jsx'),
+    TableError: ({ message }) => <div>{message}</div>,
+}));
+
+jest.mock('@deriv/shared/src/utils/screen/responsive', () => ({
+    ...jest.requireActual('@deriv/shared/src/utils/screen/responsive'),
+    isDesktop: jest.fn(),
+}));
+
+const setItems = jest.fn();
+const setIsLoading = jest.fn();
+const loadMoreItems = jest.fn();
+
+const mocked_buy_sell_store = {
+    api_error_message: '',
+    has_more_items_to_load: false,
+    is_loading: false,
+    items: [],
+    loadMoreItems,
```

Comment:
@artsemkavaleu-deriv believe that we can assign jest.fn() here (in mocked_buy_sell_store) instead of creating new variables in 27,28,29 strings
loadMoreItems: jest.fn(),
setIsLoading: jest.fn(),
loadMoreItems: jest.fn(),

Reply:
> @artsemkavaleu-deriv believe that we can assign jest.fn() here (since we don't use these methods in expect() functions) instead of creating new variables in 27,28,29 strings
> 
> ```
> setItems: jest.fn(),
> setIsLoading: jest.fn(),
> loadMoreItems: jest.fn(),
> ```

yes, since we don't need to check function calls in tests, we can define them inside that object, thx for notice!